primitives/commitment/src/piece.rs

```diff
@@ -100,6 +100,15 @@ impl Into<filecoin_proofs::UnpaddedBytesAmount> for UnpaddedPieceSize {
     }
 }
 
+#[derive(Debug, Eq, PartialEq)]
+pub enum PaddedPieceSizeError {
+    SizeTooSmall,
+    SizeNotPowerOfTwo,
+    NotAMultipleOfNodeSize,
+    InvalidPieceCid,
```

Comment:
These two variants should be removed. The piece size has no relevance with the piece cid.

Reply:
You'll need to be more specific 
![image](https://github.com/user-attachments/assets/d471a232-29fa-4f76-9461-26954434557b)